src/Node/ClassStatementsGatherer.php

```diff
@@ -158,6 +162,36 @@ private function gatherNodes(Node $node, Scope $scope): void
 			$this->tryToApplyPropertyReads($node, $scope);
 			return;
 		}
+
+		if ($node instanceof Expr\FuncCall && $node->name instanceof Node\Name && $this->reflectionProvider->hasFunction($node->name, $scope)) {
+			$function = $this->reflectionProvider->getFunction($node->name, $scope);
```

Comment:
I am pretty sure, you want us doing this for all cases of `CallLike`. I implemented it for now only for `FuncCall` to get the first round of feedback.